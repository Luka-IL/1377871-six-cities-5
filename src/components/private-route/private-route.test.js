import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import {PrivateRoute} from "./private-route";
import React from "react";

it(`PrivateRoute render correctly when user no auth`, () => {
  const tree = renderer
      .create(
          <BrowserRouter>
            <PrivateRoute
              authorizationStatus={`NO_AUTH`}
              exact={true}
              path={`/favorites`}
              render={() => {}}
            />
          </BrowserRouter>
      ).toJSON();

  expect(tree).toMatchSnapshot();
});
