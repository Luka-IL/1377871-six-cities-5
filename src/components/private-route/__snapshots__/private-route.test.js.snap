// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute render correctly when user no auth 1`] = `null`;
